sickbeard/providers/newznab.py

```diff
@@ -162,22 +164,25 @@ def _get_episode_search_strings(self, ep_obj, add_string=''):
 
         if not ep_obj:
             return [params]
+        
+        episode = 'ep' if self.name <> "Usenet-Crawler" else 'episode'
```

Comment:
This is not unique to Usenet-Crawler, I encounter the same behavior wih nzb.su; could possibly apply to others - maybe looping through a list of providers would be a better choice than setting a single provider string